core/modules/utils/dom/modal.js

```diff
@@ -212,7 +212,7 @@ Modal.prototype.display = function(title,options) {
 	bodyWidgetNode.addEventListener("tm-close-tiddler",closeHandler,false);
 	footerWidgetNode.addEventListener("tm-close-tiddler",closeHandler,false);
 	// Whether to close the modal dialog when the mask (area outside the modal) is clicked
-	if(variables.maskClosable === "yes" || variables.maskClosable === "true") {
+	if(tiddler.fields && (tiddler.fields["mask-closable"] === "yes" || tiddler.fields["mask-closable"] === "true")) {
```

Comment:
@Jermolene .. Is it needed to check for `.fields` in the `tiddler` object. I think it's not, since every tiddler has to have a title field. So it should be redundant to check for fields here. ??

Reply:
You know, people are good at copying code...  
 

So actually I copied it from here:

https://github.com/Jermolene/TiddlyWiki5/blob/6f225cef634f8940410e92d04753ed4821d83a5a/core/modules/utils/dom/modal.js#L69

Reply:
I don't know why it does this check, so I decided to do the same.

Reply:
The vercel perview is so convenient.

Reply:
Thanks @linonetwo I think it is right to be consistent here.